Iterable-iOS-SDK/IterableDeeplinkManager.m

```diff
@@ -96,6 +109,16 @@ - (void)URLSession:(NSURLSession *)session
         completionHandler:(void (^)(NSURLRequest *))completionHandler
 {
     deepLinkLocation = request.URL.absoluteString;
+    NSArray<NSHTTPCookie *> *cookies = [NSHTTPCookie cookiesWithResponseHeaderFields:redirectResponse.allHeaderFields forURL:redirectResponse.URL];
+    for (NSHTTPCookie *cookie in cookies) {
+        if ([cookie.name isEqualToString:@"iterableEmailCampaignId"]) {
```

Comment:
Are these the same cookies that we set for attribution on web?

Reply:
yup